addons/mail/models/mail_template.py

```diff
@@ -527,6 +527,8 @@ def _generate_template(self, res_ids, render_fields,
 
     @classmethod
     def _parse_partner_to(cls, partner_to):
+        if partner_to[:1] == '[' and partner_to[-1:] == ']':
```

Comment:
That's a bit low-level :) would be better to use ``literal_eval`` (probaby in a try / except statement). Playing with it gives something like
```
In [1]: from ast import literal_eval as le
In [2]: le("[1, 2, 3]")
Out[2]: [1, 2, 3]
In [3]: le("1, 2, 3")
Out[3]: (1, 2, 3)
In [4]: le("1, 2, 3,4, False")
Out[4]: (1, 2, 3, 4, False)
In [5]: le("res.partner(1, 2, 3)")
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
[...]
ValueError: malformed node or string on line 1: <ast.Call object at 0x7fe71b87dc00>
``

Reply:
+ unit test would be welcomed as usual :)